frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -320,6 +321,7 @@ export const createResources = async (
     serverless: { scaling },
     route,
   } = formData;
+  const imageStreamName = imageStream && imageStream.metadata.name;
```

Comment:
can we have `_.get(imageStream, 'metadata.name')`

Reply:
fixed in https://github.com/openshift/console/pull/2137/commits/f88e7a70a6d69715df9b91e3ef0ccf7c990c6429